src/classes/ACCT_IndividualAccounts_TDTM.cls

```diff
@@ -122,6 +125,11 @@ public class ACCT_IndividualAccounts_TDTM extends TDTM_Runnable {
         this.dmlWrapper = new DmlWrapper();
         contactsSettings = UTIL_CustomSettingsFacade.getContactsSettings();
 
+        if (triggerAction == TDTM_Runnable.Action.BeforeInsert ||
```

Comment:
Contextually, I'm understanding here that we infer that any time this individual account trigger is in before insert or after insert context, we are treating it as a trigger that is .... firing as the result of a new contact insert? And maybe the logic being that there's never a situation where we are creating an individual account other than a direct result of adding a new contact. 

It still feels a bit 'off' somehow, for any future situation where somehow we might be in this trigger context without it resulting from a new contact insert. But I also might be mis-reading the intent here? 

I wish you could look back to the sobject type of the DML that kicked off the chain of triggers that got us here or something, or get better info about the sequence of events. Triggers are - I believe - designed to fire largely agnostic of this broader context, and here we are trying to make an educated guess about the initial event that got us here.